public/index.html

```diff
@@ -21,10 +21,10 @@
             <p>{{{description}}}</p>
       </article>
     </script>
-    <script id="gitTemplate" type="text/x-handlebars-template" >
+    <script type="text/x-handlebars-template" id="gitTemplate">
       {{#each repos}}
         <li>
-          <h3>{{this.name}}</h3>
+          <h3><a href="/repos/{{this.name}}">{{this.name}}</a></h3>
```

Comment:
You have no route called repos, which is causing your page to break when you click on this. Adding this to your routes, with the right route name here and accesses the :names would cause your site not to break when